beancount/ingest/extract_test.py

```diff
@@ -18,10 +18,7 @@
 from beancount.ingest import extract
 from beancount.ingest import importer
 from beancount.ingest import scripts_utils
-
-
-extract_main = functools.partial(scripts_utils.trampoline_to_ingest, extract)
-
+from beancount.ingest.test_utils import TestScriptsBase, TestExamplesBase
```

Comment:
Favor importing package, not symbols within, most of the time,.

Reply:
Noted. I don't recall why I wrote this import like this.